tests/css_comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -96,14 +96,26 @@ exports[`selector.css 1`] = `
 .powerPathNavigator table.powerPathInfo th:active,
 .powerPathNavigator table.powerPathInfo th:active + th:last-child {
 }
+
+/* pressed buttons */
+.powerPathNavigator .helm button.pressedButton,
+.powerPathNavigator .helm button:active:not(.disabledButton) {
+}
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .powerPathNavigator .helm button.pressedButton,
 /* pressed buttons */
-.powerPathNavigator .helm button:active:not(.disabledButton),
+  .powerPathNavigator
+  .helm
+  button:active:not(.disabledButton),
```

Comment:
This is the test that broke. Prettier thinks that the comment `/* pressed buttons */` belongs to the same group as the rest of the selectors. How should we handle this case, @vjeux?